docs/en/wallet-attestation.rst

```diff
@@ -458,7 +458,7 @@ The body of the Wallet Attestation Request JWT MUST contain:
       - JSON object with name/value pairs, identifying a Credential format supported by the Wallet.
       -
     * - **authorization_endpoint**
-      - URL of the Wallet Authorization Endpoint (custom url schema or universal link of the Wallet Instance).
+      - URL of the Wallet Authorization Endpoint, preferably using a universal link for enhanced security and fallback support, though a custom url-scheme can also be used if necessary.
```

Comment:
```suggestion
      - URL of the Wallet Authorization Endpoint, it can be a universal link or a custom url-scheme.
```

we do not repeat the same words already wrote few lines above.